src/editor/components/Main.js

```diff
@@ -201,6 +210,7 @@ export default class Main extends Component {
     const scene = this.state.sceneEl;
     const isEditor = !!this.state.inspectorEnabled;
     const sceneData = AFRAME.scenes[0].getAttribute('metadata', 'sceneTitle');
+    const isProUser = currentUser && currentUser.isBeta;
```

Comment:
That should be currentUser.isPro or isBeta && isPro maybe?

Reply:
I thought that we are still using isBeta flag, while Pro mode is not in production? as @kfarr said. Then I will need to change isBeta to isPro

Reply:
fixed